### DISCUSSION TITLE
error: "name 'C_mrg' is not defined"

### DISCUSSION THREAD

**Discussion by tom221085** [miniscope/minian#142] [Q&A] [answered]
Hello,

since last week I am getting the following error message:
![MinianError1](https://user-images.githubusercontent.com/11939642/121917399-153f5480-cd03-11eb-8b3b-58a6b9dd982c.PNG)

and then this error in the following cell:
![MinianError2](https://user-images.githubusercontent.com/11939642/121917473-28eabb00-cd03-11eb-9dd5-1ca6ab579cbc.PNG)

I tried to figure out if it's because of a parameter change but if I set everything back to default I still get this error, no matter which data I am using.

Cheers,
Thomas

**Comment by phildong** [comment, marked as answer]
It seems for some reason there is no overlap between the spatial footprints at all, so the algorithm doesn't have anything to merge. Can you confirm this?

**Reply by tom221085** [reply]
Yes, that seems to be the issue. I'll try different parameters.
